.github/workflows/python-integration-test.yml

```diff
@@ -57,18 +57,26 @@ jobs:
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
 
-      - name: Free up disk space
-        run: |
-          dev/ci/util_free_space.sh
-
       - name: Python Client Integration Test
         id: integrationTest
+        env:
+          EXTERNAL_START_GRAVITINO: true
         run: |
           ./gradlew compileDistribution -x test -PjdkVersion=${{ matrix.java-version }}
+
           for pythonVersion in "3.8" "3.9" "3.10" "3.11"
           do
-            ./gradlew -PjdkVersion=${{ matrix.java-version }} -PpythonVersion=${pythonVersion} :client:client-python:integrationTest
+            echo "Use Python version ${pythonVersion} to test the Python client."
+            # Start Gravitino server and set the environment variable
+            ./distribution/package/bin/gravitino.sh start
```

Comment:
With this manual `startUp` step, how do we test in local environment using gradle?

Reply:
1. Test Principle: Every Python ITs class base on the `IntegrationTestEnv`, `IntegrationTestEnv` will automatic start and stop Gravitino server to support ITs, But when you run multiple ITs class at same time, The first test class that finishes running will shut down the Gravitino server, which will cause other test classes to fail if they can't connect to the Gravitino server.
2. Run test in the IDE: Through `IntegrationTestEnv` class automatic start and stop Gravitino server to support ITs.
3.  Run test in the Github Action: Manual start and stop Gravitino server, and set `EXTERNAL_START_GRAVITINO` environment variable
4. Run test in the Gradle command `:client:client-python:test`: Gradle automatic start and stop Gravitino server, and set `EXTERNAL_START_GRAVITINO` environment variable.

I will create a document to description this in the next PR.